features/Context/FieldTypeFormContext.php

```diff
@@ -144,23 +145,120 @@ public function theFieldDefinitionIsMarkedAsRequired()
+     */
+    public function theSelectFieldShouldBeFlaggedAsRequired()
+    {
+        $this->theFieldShouldBeFlaggedAsRequired('select');
+    }
+
+    /**
+     * Assert that a form field should be flagged as required.
+     *
+     * @param $elementType string The type of form element (e.g. 'input', 'select')
+     */
+    protected function theFieldShouldBeFlaggedAsRequired($elementType)
+    {
+        $nodeElements = $this->getSession()->getPage()->findAll(
             'css',
-            sprintf('div.ezfield-identifier-%s fieldset input', self::$fieldIdentifier)
+            sprintf('div.ezfield-identifier-%s fieldset %s', self::$fieldIdentifier, $elementType)
         );
 
-        Assertion::assertNotEmpty($inputNodeElements, 'The input field is not marked as required');
+        Assertion::assertNotEmpty($nodeElements, "The $elementType field is not marked as required");
+
+        foreach ($nodeElements as $nodeElement) {
+            $failureMessage = trim(sprintf(
+                '%s %s with id %s is not flagged as required',
+                $nodeElement->hasAttribute('type') ? $nodeElement->getAttribute('type') : '',
+                $elementType,
+                $nodeElement->getAttribute('id')
+            ));
 
-        foreach ($inputNodeElements as $inputNodeElement) {
             Assertion::assertEquals(
                 'required',
-                $inputNodeElement->getAttribute('required'),
-                sprintf(
-                    '%s input with id %s is not flagged as required',
-                    $inputNodeElement->getAttribute('type'),
-                    $inputNodeElement->getAttribute('id')
-                )
+                $nodeElement->getAttribute('required'),
+                $failureMessage
             );
         }
     }
+
+    /**
+     * Set a boolean field definition option $option to $value.
+     *
+     * @param $option string The field definition option
+     * @param $value bool The option value
+     */
+    protected function setBooleanFieldDefinitionOption($option, $value)
+    {
+        $this->contentTypeContext->updateFieldDefinition(
+            self::$fieldIdentifier,
+            new FieldDefinitionUpdateStruct(['fieldSettings' => [$option => $value]])
+        );
+    }
+
+    /**
+     * @Then the input is a single selection dropdown
+     */
+    public function theInputIsASingleSelectionDropdown()
+    {
+        $this->assertElementOnPage(
+            sprintf(
+                'div.ezfield-identifier-%s fieldset select',
+                self::$fieldIdentifier
+            )
+        );
+        $this->assertElementNotContains(
+            sprintf(
+                'div.ezfield-identifier-%s fieldset select',
+                self::$fieldIdentifier
+            ),
+            'multiple="multiple"'
+        );
+    }
+
+    /**
+     * @Then the input is a multiple selection dropdown
+     */
+    public function theInputIsAMultipleSelectionDropdown()
```

Comment:
If we decide to use dedicated context for dedicated FieldTypes, maybe these should go to the `SelectFieldTypeFormContexts` ?